src/haskell.rs

```diff
@@ -0,0 +1,37 @@
+use npezza93_tree_sitter_tags::{Tag as TSTag, TagsContext, TagsConfiguration};
```

Comment:
Note: this file is almost identical to `src/rust.rs`.
It might make sense to extract `generate_tags` into a strategy.